gradle.properties

```diff
@@ -1,27 +0,0 @@
-# Project-wide Gradle settings.
```

Comment:
why was this deleted?

Reply:
I think when I added added new changes to git I added some unnecessary files too, so while resetting the added changes I think it got deleted.
Fixing this right away